File: C/String/program21.c
//Accept the string from user and copy that string into another string

#include<stdio.h>

void strcpyX(char *src, char *dest)
{
    while(*src != '\0')
    {
        *dest = *src;
        src++;
        dest++;
    }
    *dest = '\0';
}

int main()
{
    char Arr[20];
    char Brr[20];

    printf("Enter string: \n");
    scanf("%[^'\n']s",Arr);

    strcpyX(Arr,Brr);
    printf("String after copy : %s ",Brr);

    return 0;
}

File: Java/String/program21.java
//Accept the string from user and toggle that string 

import java.util.*;

class Stringx
{
    public String str;

    public void Accept()
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter the string");
        str = sobj.nextLine();
    }

    public void strtoggleX()
    {
        char Arr[] = str.toCharArray();

        int i = 0;
        int ascii = 0;
        char ch = '\0';

        for(i = 0; i < Arr.length; i++)
        {
            if( (Arr[i] >= 'A') && (Arr[i] <= 'Z'))
            {  
                ch = Arr[i];
                ascii = ch;
                ascii = ascii + 32;
                ch = (char)ascii;
                Arr[i] = ch;
            }
            else if( (Arr[i] >= 'a') && (Arr[i] <= 'z'))
            {  
                ch = Arr[i];
                ascii = ch;
                ascii = ascii - 32;
                ch = (char)ascii;
                Arr[i] = ch;
            }
        }

        System.out.print("String after toggle : ");

        for(i = 0; i < Arr.length; i++)
        {
            System.out.print(Arr[i]);
        }        
    }
}

class program21
{
    public static void main(String arg[])
    {
        Stringx obj = new Stringx();

        obj.Accept();

        obj.strtoggleX();
    }
}